partner_contact_tag/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Contact Tag",
+    "summary": """
+        Display tag in the pop-up form view when adding a contact to a partner.""",
+    "version": "16.0.1.0.0",
+    "category": "Uncategorized",
```

Comment:
most modules in [OCA/partner-contact](https://github.com/OCA/partner-contact) seems to use the "Customer Relationship Management" category.